README.md

```diff
@@ -37,16 +36,89 @@ Install-Module Pester -Scope CurrentUser
 ### Install from Gallery
 
 1. `Install-Module PSKoans -Scope CurrentUser`
-2. Run `Measure-Karma` to get things started.
-3. Run `Measure-Karma -Meditate` to open your Koans folder either directly or in VS Code (if installed).
 
-### Clone the Repo
+### Or clone the Repo
 
 1. `git clone` the repository into your desired directory, or download the repository as a .zip file and extract into a directory of your choice.
 2. Then from a normal powershell session run `Get-ChildItem -Recurse | Unblock-File` in that directory to remove the "downloaded from internet" flag that blocks them from running.
 3. Check `Get-ExecutionPolicy`: if it says 'Restricted' or 'Undefined', you need to also run `Set-ExecutionPolicy RemoteSigned` in order to allow the scripts to run.
 4. Run `Install-Module -Path .\Path\To\PSKoans-master`
-5. Before working with the module, run `Measure-Karma` (`rake` for short) once to initialise everything, and then run `Measure-Karma -Koans` (or `-Meditate`) to open the Koans folder for you to begin your journey.
+
+### Start your Journey
+
+1. Run `Measure-Karma` to start your journey towards PowerShell enlightenment. You will be presented with a page describing your goal:
+
+```diff
+    Welcome, seeker of enlightenment.
+    Please wait a moment while we examine your karma...
+
+Describing 'Equality' has damaged your karma.
+
+-    You have not yet reached enlightenment.
+
+    The answers you seek...
+
+- Expected $true, but got $null.
+
+    Please meditate on the following code:
+
+- [It] is a simple comparison
+- at <ScriptBlock>, ...\Foundations\AboutAssertions.Koans.ps1: line 27
+- 27:         __ | Should -Be $true
+
+    Even if you speak of the wonder of it all,
+    How do you deal with each thing changing?
+
+    Your path thus far:
+
+ [―――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――] 0/238
+
+You may run 'rake -Meditate' to begin your meditation.
+```
+
+Inspect the red messages carefully, most importantly the last one. The error message contains path to the file that you need to edit in order to progress forward, in this case `Foundations\AboutAssertions.Koans.ps1`. 
+
+2. Run `Measure-Karma -Meditate` to open your Koans folder and navigate to `Foundations\AboutAssertions.Koans.ps1`. Near the top you'll see:
+
+```powershell
+It 'is a simple comparison' {
+    # Some truths are absolute.
+    __ | Should -Be $true
+}
+```
+
+The `__` represents a blank for you to fill, and `| Should -Be $true` shows the expected result. To pass this koan you need to replace `__` with `$true`, like this: `$true | Should -Be $true`.
```

Comment:
I'm not entirely convinced we should have a complete walkthrough for the first koan. Sure, it's simple as heck, but it seems a little... weird. I dunno.

Reply:
@vexx32 the koan is extremely simple, but understanding what you should do to take on this journey is not.

Imagine you don't know anything about powershell, pester, testing or koans. Then the first page you see offers no real info, because you don't know what Describe, It or Should is. You don't know that you are looking at a mix of standard PowerShell messages, Pester messages, and custom stuff.

So you try the second command which opens vscode without pointing you to any file. And then you just click around lost. Like I did this morning. 😄 

So if it took me good 10 minutes (and two visits to google) to understand what you want me to do, then for a beginner it won't be much easier. I admit I scrolled directly to getting started on the github readme, and if I did not do it I might have had easier time, but that's what people do, they skim stuff. Which is one of the reasons I included the whole screens with some colors, so they anchor your vision to something similar to what you see on the screen.

Reply:
Yeah, that's fair enough, haha.

Thank you for this! 😄